### PR TITLE
Add step for bumping packages when doing RC patch

### PR DESCRIPTION
**Pull request by fortmarek**
When doing an RC patch, we should run `yarn bump-all-updated-packages` to ensure that we release the changed packages with the new RC.

### CODE REVIEW

**Review by cortinico — APPROVED** [review]

File: website/contributing/release-candidate-patch.md [hunk 1]
```diff
@@ -29,13 +29,19 @@ git checkout -b <release-branch>
 git cherry-pick <commit-hash>
 ```
 
-### 2. Test the current changes
+### 2. Bump monorepo packages
+
+- Update packages in the monorepo that were modified by the cherry picks by running `yarn bump-all-updated-packages`. All the package bumps should be on the patch level. Read more about the script and how they work [here](./release-updating-packages).
+
+- CI will take care of releasing the new versions on npm once you push the changes to the `0.XX-stable` branch.
```
Comment (kelset) [inline website/contributing/release-candidate-patch.md:36]: quick note (already mentioned on the side): remember is that in 0.71 it's 2 separate commands, you need to run the bumping one first, then push it up and wait for the packages to get released, then do the alignment one, while in 0.72 it's one command and you just push it after.

(and well then in 0.70 and lower there's nothing 😅)

Comment (cipolleschi) [inline website/contributing/release-candidate-patch.md:36]: Following the first point, I'll be more prescriptive here:

```suggestion
Update all packages in the monorepo that were modified by the cherry picks. You can do it by running:
    ```
    yarn bump-all-updated-packages # All the package bumps should be on the patch level
    git push origin 0.XX-stable
    ```
After pushing, the CI will take care to publish the new packages automatically.

**For 0.71: **
// Add instructions here

** For 0.70: **
For version 0.70, packages must be released manually by a Meta engineer. 
```

**Review by cipolleschi — CHANGES_REQUESTED** [review]

File: website/contributing/release-candidate-patch.md (hunk 1 above)
Comment (fortmarek) [reply website/contributing/release-candidate-patch.md]: We won't release an RC for 0.71 and 0.70, so I skipped those steps in this document. However, I added them to the `stable-patch` which we will do for those versions.

File: website/contributing/release-stable-patch.md [hunk 2]
```diff
@@ -27,13 +27,42 @@ git pull origin <release-branch> --tags
 git cherry-pick <commit>
 ```
 
-### 2. Test the current changes
+### 2. Bump monorepo packages
```
Comment (fortmarek) [inline website/contributing/release-stable-patch.md:30]: @kelset can you double check the new steps are correct both for 0.72 and 0.71? 🙏

File: website/contributing/release-stable-patch.md [hunk 3]
```diff
@@ -27,13 +27,42 @@ git pull origin <release-branch> --tags
...
+
+**For 0.72 and higher:**
+
+Update all packages in the monorepo that were modified by the cherry picks. You can do it by running:
+
+```sh
+yarn bump-all-updated-packages # All the package bumps should be on the patch level
```
Comment (kelset) [inline website/contributing/release-stable-patch.md:37]:
```suggestion
yarn bump-all-updated-packages # All the package bumps must be on the patch level
```

**Review by kelset — COMMENTED** [review]
looks great, it's ready for merge to me.

thanks Marek!

cc @Simek

**Review by cipolleschi — APPROVED** [review]

**Pull request merged**